### PR TITLE
Pop quiz 03102021

### PR DESCRIPTION
**Pull request by billythakidd04**
I have added @pmehtaaustin and @tbonesteak as reviewers as they are independant parties.

Please pull down this branch and fill in your answers and push the **same branch** back up.

### CODE REVIEW

File: Pop_Quiz.md [hunk 1]
```diff
@@ -202,3 +210,38 @@ while(count($stuff) < 8){
...
+@gc824
+
+1. What does the `++` operator do?
+@eiman-kased
+
+1. Correct this code: `console.log('There's something wrong here!')`
+@CharleswithLove
+
+
+1. What is the advantage of using a front end programming language?
```
Comment (ghost) [inline Pop_Quiz.md:247]: To create a user interactive environment.

**Review by ghost — DISMISSED** [review]

File: Pop_Quiz.md (hunk 1 above)
Comment (billythakidd04) [reply Pop_Quiz.md:247]: You need to add your changes to the file and push them up on the same branch

File: Pop_Quiz.md [hunk 2]
```diff
@@ -225,7 +225,7 @@ echo $stuff[$counter];
 @vricci518 use the script tag, it can be placed within the head of the html file. For Bootstrap it recomended to put is just above the /body tag to the page can load first.
 
 1. What does the `%` symbol mean?
-@rbaia28
+@rbaia28 called a modulus, shows the remainder
```
Comment (billythakidd04) [inline Pop_Quiz.md]: @rbaia28 remainder of what?

**Review by billythakidd04 — COMMENTED** [review]
@rbaia28

**Comment by billythakidd04** [comment]
@cwlainson and @WaymonBrown84 I still need your answers on this please!

**Review by ghost — APPROVED** [review]

**Pull request merged**